sudo/utils.py

```diff
@@ -65,44 +60,3 @@ def has_sudo_privileges(request):
         except (KeyError, BadSignature):
             request._sudo = False
     return request._sudo
-
-
-def is_safe_url(url, host=None):
```

Comment:
Can we leave an import to `django.utils.http:is_safe_url` left in here just to not change a public API? Just leave it as an alias.

Reply:
Good idea.